extension/lib/doc.js

```diff
@@ -157,19 +157,20 @@ class Doc {
                 this.sentenceEls.push($(filteredSentenceEls));
                 // Set keywords and score for each sentence
                 this.setSentenceKeywordsAndScore(container, containerText, start, end, sentenceId);
```

Comment:
the trump page took 10 secs to load, and after that the app just says "too big, can't handle!"
I saw that max_num_sentence is 4K.

This might be lower, or.. we keep track of time elapsed so far and limit it by time in addition to num sentences.

Leavnig as clickup item

Reply:
https://app.clickup.com/t/apphrh